docs/Daos.md

```diff
@@ -0,0 +1,262 @@
+# Working with DAOs
+
+Arc.js provides a class called [DAO](api/classes/DAO) that facilitates creating and working with DAOs.  Read on for more information about DAOs in DAOstack and the [DAO](api/classes/DAO) class.
+
+## About DAOs
+
+Every DAO in DAOstack has an architecture, implemented by reusable Arc contracts, that includes an avatar, a controller, a token, a reputation system, and a set of schemes and global constraints.
+
+**The avatar** is a public-facing part of the DAO that handles the interaction of the DAO with the rest of the world, for example interacting with other DAOs or paying third party agents.  A DAO is always referenced by the address of its avatar.
+
+**The controller** is a central internal hub for all of the components of the DAO.  DAOstack documentation often refers to the various components as "organs".
```

Comment:
often refers to the various dao's components as "organs".

Reply:
Done